hack/tools/create_kind.sh

```diff
@@ -30,22 +30,33 @@ registry_create() {
 }
 
 certs_create() {
-    mkdir ../registry -p || true
-    openssl genrsa -out ../registry/ca.key 2048
-    openssl req -new -x509 -key ../registry/ca.key -out ../registry/ca.crt -subj '/C=US/ST=NY/O=IBM/CN=ibm' -extensions EXT -config <(printf "[dn]\nCN=ibm\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:ibm\nbasicConstraints=CA:TRUE,pathlen:0")
-    openssl genrsa -out ../registry/registry.key 2048
-    openssl req -new -key ../registry/registry.key -out ../registry/registry.csr -subj '/C=US/ST=NY/O=IBM/CN=kind-registry' -extensions EXT -config <(printf "[dn]\nCN=kind-registry\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:kind-registry,DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
-    openssl x509 -req -in ../registry/registry.csr -CA ../registry/ca.crt -CAkey ../registry/ca.key -CAcreateserial -out ../registry/registry.crt
+    mkdir -p ../registry || true
+
+    if [ ! -f ../registry/ca.crt ]; then
+      openssl genrsa -out ../registry/ca.key 2048
+      openssl req -new -x509 -key ../registry/ca.key -out ../registry/ca.crt -subj '/C=US/ST=NY/O=IBM/CN=themeshfordata' -extensions EXT -config <(printf "[dn]\nCN=ibm\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:ibm\nbasicConstraints=CA:TRUE,pathlen:0")
+    fi
+    if [ ! -f ../registry/registry.crt ]; then
+      openssl genrsa -out ../registry/registry.key 2048
+      openssl req -new -key ../registry/registry.key -out ../registry/registry.csr -subj '/C=US/ST=NY/O=IBM/CN=kind-registry' -extensions EXT -config <(printf "[dn]\nCN=kind-registry\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:kind-registry,DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
+      openssl x509 -req -in ../registry/registry.csr -CA ../registry/ca.crt -CAkey ../registry/ca.key -CAcreateserial -out ../registry/registry.crt
+    fi
 }
 
 install_certs() {
     if [[ "$OSTYPE" == "linux-gnu"* ]]; then
         sudo cp ../registry/ca.crt /usr/local/share/ca-certificates
```

Comment:
Not related to this PR but I think it isn't true for all linux machines (e.g., RHEL uses a different folder + binary). Need to at least document this.

Reply:
I assumed that

> The vendoring is because we want to set the local environment variable at some point for development. When using the all-in-one.yaml from razee it downloads the deployments via RemoteResource, which is nice because you get updates but if you change your deployment manually (e.g. to set the local authentication environment variable) it's also bad because it will overwrite your manually adapted deployment with what is defined remotely.
> So for now I thought the easiest is to use the yamls so we can adapt them instead of pulling them from razee. Another solution would be to use a templated razee version if that is availabe.

I think it's worth asking on the Razee slack channel if there is a better approach than vendoring it. But okay for now.